lang/system/pl.xml

```diff
@@ -55,9 +114,12 @@ Kod błędu: {}</an_error_occurred>
 		<file_corrupted>Plik jest uszkodzony.</file_corrupted>
 		<microphone_disabled>Włącz mikrofon.</microphone_disabled>
 		<no>Nie</no>
+		<ok>OK</ok>
```

Comment:
no need add it, is same exist in English.

Reply:
As it's updated God's know when, it was easier to follow and compare with English file, that will be always up to date.

Reply:
Done.